apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -14,7 +14,8 @@
          AssetsHelper,
          SMIL,
          TaskRunner,
-         Thread
+         Thread,
+         ActivityClient
```

Comment:
same here, put ActivityClient before AssetsHelper (trying to have them alphabetically organized for the new ones...); you can move AssetsHelper before DocumentFragment too...